recipes-aos/aos-communicationmanager/files/aos-communicationmanager.service

```diff
@@ -2,7 +2,7 @@
 Description=AOS Communication Manager
 Wants=network-online.target aos-iamanager.service
 After=network-online.target aos-iamanager.service
-ConditionPathExists=!/var/aos/.unprovisioned
+ConditionPathExists=/var/aos/.provisionstate
```

Comment:
should we check content of the file as well?

Reply:
no, absence of the .provisionstate file means node in the unprovisioned state